### ISSUE TITLE
[BUILD]Typo : InstallDotNet call twice GetDotNetInstallScript

### ISSUE THREAD

**Issue by anurse** [dotnet/arcade#3304]
_From @tebeco on Wednesday, July 17, 2019 1:19:05 PM_

### Describe the bug
At the end of the line, it seems that an manual indent or reformat ended up in a duplicate call to `GetDotNetInstallScript` : 

Scroll to the right of this code block :
```powershell
function InstallDotNet([string] $dotnetRoot, [string] $version, [string] $architecture = "", [string] $runtime = "", [bool] $skipNonVersionedFiles = $false) {  $installScript = GetDotNetInstallScript $dotnetRoot
  $installScript = GetDotNetInstallScript $dotnetRoot
  $installParameters = @{
```
### Expected behavior
only one call should be made to `GetDotNetInstallScript` in `InstallDotNet`

#### Side notes 
It's like this from the very first commit,
it looks like it was created by either an automation or from an already existing script, so this may be existing in another repo somewhere publicly or internaly
https://github.com/aspnet/AspNetCore/commit/347ddcb6e20a5ef291972604407894baf5f14526#diff-58d6d2c2760628d9551cc8b40461ce53

_Copied from original issue: aspnet/AspNetCore#12275_

**Comment by anurse** [comment]
_From @tebeco on Wednesday, July 17, 2019 2:52:56 PM_

Sorry for pinging you like this, it seems this time github did not auto assigned anyone, just saw you were involved in other build issues
@natemcmaster 
@mkArtakMSFT
@anurse 

After investigating weird behavior on my box i found this.

**Comment by anurse** [comment]
_From @rynowak on Wednesday, July 17, 2019 2:53:57 PM_

@aspnet/build

**Comment by anurse** [comment]
_From @anurse on Wednesday, July 17, 2019 3:04:42 PM_

This is actually an arcade issue, as this content comes from dotnet/arcade. Moving.

**Comment by anurse** [comment]
cc @dougbu @JunTaoLuo (to keep them in the look since you can't mention teams cross-org it seems)

**Comment by tebeco** [comment]
(i'm attempting right now to re-create the PR for this repo)

**Comment by tebeco** [comment]
done

**Comment by markwilkie** [comment]
@dougbu - could you elaborate more what's happening here please?

**Comment by dougbu** [comment]
The problem was a typo where `$installScript = GetDotNetInstallScript $dotnetRoot` appeared at the end of the first line @tebeco quoted in the description as well as on the second line. We should have closed this issue when @jonfortescue merged #3305.
